src/table.ts

```diff
@@ -795,16 +789,17 @@ class Table {
   private _mutate(
     method: string,
     rows: object | object[],
-    callback: s.CommitCallback
+    callback: CommitCallback
   ): void {
     this.database.runTransaction((err, transaction) => {
       if (err) {
         callback(err);
         return;
       }
 
-      transaction[method](this.name, rows);
-      transaction.commit(callback);
+      // tslint:disable-next-line: no-any
+      (transaction as any)![method](this.name, rows);
```

Comment:
can you try without `any`?
```ts
transaction![method](this.name, rows);
```

Reply:
while enable NoImplicitAny it gives an error.

Reply:
ok